client/my-sites/email/email-management/home/email-plan-subscription.jsx

```diff
@@ -13,6 +14,19 @@ import AutoRenewToggle from 'calypso/me/purchases/manage-purchase/auto-renew-tog
 import { withLocalizedMoment } from 'calypso/components/localized-moment';
 
 class EmailPlanSubscription extends React.Component {
+	isSubscriptionExpired() {
```

Comment:
I have the feeling that `hasSubscriptionExpired` would be more accurate/valid but I'm not a native English speaker.

Reply:
You are 100% correct -- I'll update to `hasSubscriptionExpired`!